### ISSUE TITLE
Improved documentation: JSON.Get

### ISSUE THREAD

**Issue by W-Lawless** [michaelvanstraten/swifty-redis#3]
I implemented JSON.Get with decoder as reusable generic function as follows: 

```
final class RedisService { 

var connection: RedisConnection? 

func connect() async { . . . }

func getJSON<T: FromRedisValue>(key: String) async -> T? {
        if let connection {
            do {
                print("Fetching JSON")
                let value: T = try await Cmd("JSON.GET").arg(key).query(connection)
                return value
            } catch {
                print(error)
                return nil
            }
        } else {
            return nil
        }
    }
}
```

Relevant JSON decoder type:

```
struct FrontEndConfig: FromRedisValue {
    
    var value: FrontEndModel? = nil
    let decoder = JSONDecoder()
    
    init(_ value: SwiftyRedis.RedisValue) throws {
        switch(value) {
        case .BulkString(let response):
            let data = Data(response.utf8)
            let decoded = try decoder.decode(FrontEndModel.self, from: data)
            self.value = decoded
        default:
            break;
        }
    }
}

struct FrontEndModel: Codable {
    var fields: [UIField]
}

struct UIField: Codable, Identifiable {
    var id: String
    var name: String
}
```

at call site 

```
let testJSON: FrontEndConfig? = await redisService.getJSON(key: "frontend:Properties")
```

**Comment by michaelvanstraten** [comment]
Is this a feature request, if yes, is a great addition to the package that could be useful in many cases. 

But maybe instead of a function, we could define a wrapper type that implemented the `FromRedisValue` and `ToRedisArgs` protocols. This way, using this example, `FronEndConfig` would only have to implement `Codable` and `Decodable`.

**Comment by W-Lawless** [comment]
Not exactly a feature request, I figured you could just add it to your docs as example boilerplate. This is just how I implemented it as within my service class, but if you could work it into the library more naturally to reduce some verboseness, that would be great too!

**Comment by michaelvanstraten** [comment]
Solved in ecc1e3a
